src/Commands/StartCommand.php

```diff
@@ -31,23 +32,10 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $config = $this->initializeConfig($input, $output);
+        $class = $config->getProcessManager();
```

Comment:
getProcessManagerClass? Change options name, too??

Reply:
Honestly I'm wondering why the option is even a thing, considering PM has a ton of responsibilities, and changes like this will completely break those custom implementations if they attempt to update. Is there a reason to use this setting?

Reply:
I think that was my fault. Remove it from the game by (separate) PR? I can't imagine that it has even been used by anybody.